cigaradvisor/scripts/util.js

```diff
@@ -23,13 +23,18 @@ function createEllipsis() {
   return listItem;
 }
 
+export function scrollTop() {
+  window.scrollTo({ top: 0, behavior: 'smooth' });
```

Comment:
Do we want to scroll to the top of the window, or top of the article list?

Reply:
Good point, looped customer https://adobe-dx-support.slack.com/archives/C060KEF7DH9/p1711113617605649?thread_ts=1711055589.638429&cid=C060KEF7DH9